Scripts/script-FetchFromInstance.yml

```diff
@@ -35,8 +35,9 @@ script: |-
 
       for inc in response:
           contents = inc.get('Contents', '')
-          if 'Error' in str(contents):
-              return_error("Error occurred while fetching incidents from {0}: {1}".format(instance_name, str(contents)))
+          error_msg_in_incident = demisto.args().get('error_msg_in_incident')
+          if error_msg_in_incident and error_msg_in_incident in str(contents):
```

Comment:
Not 100% its the way to go, this way you will miss any error that may come up during the fetch.
We need to think of a solution that keeps the old logic working but with the option to bypass it in certain cases

Reply:
The main issue is that the current logic doesn't really find errors. It looks for the string `Error` in the fetch-incidents, but an incident can have the field Error in it even if it's not an error - such as in the case of this integration